internal/receiver/lightprometheusreceiver/scraper.go

```diff
@@ -34,20 +34,49 @@ import (
 )
 
 type scraper struct {
-	client   *http.Client
-	settings component.TelemetrySettings
-	cfg      *Config
-	name     string
+	client            *http.Client
+	settings          component.TelemetrySettings
+	cfg               *Config
+	name              string
+	excludeCounters   bool
+	excludeGauges     bool
+	excludeSummaries  bool
+	excludeHistograms bool
+	excludeUntyped    bool
 }
 
 func newScraper(
 	settings receiver.CreateSettings,
 	cfg *Config,
 ) *scraper {
+	excludeCounters := false
+	excludeGauges := false
+	excludeSummaries := false
+	excludeHistograms := false
+	excludeUntyped := false
+	for _, f := range cfg.MetricFamiliesExcluded {
```

Comment:
nit: as per terminology this is MetricMetadata_Type and not a MetricFamily which is of type string imo